From d6000b678fa4be9e97fa747a3b70a790bda68544 Mon Sep 17 00:00:00 2001
From: ZhongWen <b47623@freescale.com>
Date: Thu, 9 Jan 2014 11:02:24 +0800
Subject: [PATCH 055/115] ENGR00293590 Fix OTA Part new version description
 cut when there is new upgrade

Fix OTA Part new version description cut when there is new upgrade.

Signed-off-by: ZhongWen <b47623@freescale.com>
---
 FSLOta/res/layout/main.xml | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/FSLOta/res/layout/main.xml b/FSLOta/res/layout/main.xml
index 2bc7013..820dc06 100644
--- a/FSLOta/res/layout/main.xml
+++ b/FSLOta/res/layout/main.xml
@@ -28,11 +28,11 @@
 
         <TextView
             android:id="@+id/version_text_view"
-            android:layout_width="match_parent"
-            android:layout_height="50dp"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
             android:layout_column="2"
             android:layout_columnSpan="5"
-            android:layout_gravity="bottom"
+            android:layout_gravity="bottom|center"
             android:layout_row="1"
             android:layout_rowSpan="2" />
 
@@ -95,4 +95,4 @@
         android:layout_height="match_parent"
         android:layout_gravity="center_horizontal"
         android:orientation="vertical" >
-</LinearLayout>
\ No newline at end of file
+</LinearLayout>
-- 
1.8.0

